pkgs/applications/graphics/avocode/default.nix

```diff
@@ -0,0 +1,99 @@
+{ stdenv, lib, makeDesktopItem
+, gdk_pixbuf, glib, gtk2, atk, pango, cairo, freetype, fontconfig, dbus, nss, nspr, alsaLib, cups, expat, udev
+, libX11, libxcb, libXi, libXcursor, libXdamage, libXrandr, libXcomposite, libXext, libXfixes, libXrender, libXtst, libXScrnSaver
+, gnome2
+}:
+
+stdenv.mkDerivation rec {
+  name = "avocode-${version}";
+  version = "2.26.0";
+
+  src = fetchurl {
+    url = "https://media.avocode.com/download/avocode-app/${version}/avocode-${version}-linux.zip";
+    sha256 = "11d3nlshyzh5aqf5vsvnjwhr9qn8a2kd848x0ylv91y9p9njgsl5";
+  };
+
+  libPath = stdenv.lib.makeLibraryPath [
+    stdenv.cc.cc.lib
+    gdk_pixbuf
+    glib
+    gtk2
+    atk
+    pango
+    cairo
+    freetype
+    fontconfig
+    dbus
+    nss
+    nspr
+    alsaLib
+    cups
+    expat
+    udev
+    libX11
+    libxcb
+    libXi
+    libXcursor
+    libXdamage
+    libXrandr
+    libXcomposite
+    libXext
+    libXfixes
+    libXrender
+    libXtst
+    libXScrnSaver
+    gnome2.GConf
+  ];
+
+  desktopItem = makeDesktopItem {
+    name = "Avocode";
+    exec = "avocode";
+    icon = "avocode";
+    desktopName = "Avocode";
+    genericName = "Design Inspector";
+    categories = "Application;Development;";
+    comment = "The bridge between designers and developers";
+  };
+
+  dontBuild= true;
+  buildInputs = [unzip];
+  nativeBuildInputs = [ makeWrapper ];
+
+  buildCommand = ''
```

Comment:
Why use `buildCommand` instead of the standard phases? You already set `dontBuild = true`, so you may as well describe an `installPhase` and a `postFixup`.